packages/thirdweb/src/utils/shortenLargeNumber.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Shorten the string for large value
+ * Mainly used for
+ * Examples:
+ * 10_000 -> 10k
+ * 1_000_000 -> 1M
+ * 1_000_000_000 -> 1B
+ * @example
+ * ```ts
+ * import { shortenLargeNumber } from "thirdweb/utils";
+ * const numStr = shortenLargeNumber(1_000_000_000, )
+ * ```
+ * @utils
+ */
+export function shortenLargeNumber(value: number) {
+  if (value < 10_000) {
+    return value.toString();
+  }
+  if (value < 1_000_000) {
+    return formatLargeNumber(value, 1_000, "k");
+  }
+  if (value < 1_000_000_000) {
+    return formatLargeNumber(value, 1_000_000, "M");
+  }
+  return formatLargeNumber(value, 1_000_000_000, "B");
+}
+
+/**
+ * Shorten the string for large value (over 4 digits)
+ * 1000 -> 1000
+ * 10_000 -> 10k
+ * 1_000_000 -> 1M
+ * 1_000_000_000 -> 1B
+ */
+function formatLargeNumber(
```

Comment:
can you add the international formatting to this?

like $1432 should show as $1,4232

Reply:
Added